frontend/app/src/theme/community/barbie.ts

```diff
@@ -0,0 +1,73 @@
+import type { Theme } from "../types";
+
+// Colors
+const white = "#FFFFFF"
+const pink1 = "#f29ad8";
+const pink2 = "#ee71c3";
+const pink3 = "#e448a3";
+const barbieYellow = "#f9dd9a";
+const barbieBlue = "#87ced0";
+
+const txtLight = white;
+const txt = white;
+
+const notificationAccent = barbieYellow;
+const yourMessageBackground = "#e742ad";
+const friendsMessageBackground = "#94c6c9";
+
+export function getTheme(base: Theme): Theme {
+    // it's ok to mutate the theme passed in because it's a clone
+    base.name = "barbie";
+    base.label = "Barbie";
+    base.burst = false;
+    base.bg = "radial-gradient(circle, rgba(242,154,216,1) 10%, rgba(238,113,195,1) 80%)";
+    base.bd = "#e305ad";
+    base.txt = txt;
+    base["txt-light"] = txtLight;
+    base.icon.txt = white;
+    base.icon.selected = barbieBlue;
+    base.button.bg = barbieBlue;
+    base.button.hv = barbieYellow;
+    base.button.txt = "rgba(255 255 255 / 80%)";
+    base.button.disabled = pink1;
+    base.collapsible.closed.header.txt = txt;
+    base.collapsible.open.header.arrow = white;
+    base.accent = barbieYellow;
+    base.panel.left.bg = pink3;
+    base.panel.right.modal = "#e1349a";
+    base.modal.bd = base.bd;
+    base.modal.bg = pink3;
+    base.chatSummary["bg-selected"] = "rgb(225, 52, 154 / 80%)";
```

Comment:
Suggestion: 

```
    base.chatSummary["bg-selected"] = "#e75aac";
```

Otherwise the selected chat doesn't really stand out which can be useful.